README.md

```diff
@@ -10,7 +10,16 @@ funtion multiply(x) (
 
 -- YOUR ANSWER HERE --
 
-## Explain each of the below versions of writing functions
+
+function multiply(x,y){
+let sum = x* y;
+
+return sum;
+
```

Comment:
The code would work, but a product is never a sum. So `let sum = ` should be `let product = ...`